components/board.drugconnectivity/R/drugconnectivity_plot_actmap.R

```diff
@@ -0,0 +1,176 @@
+##
+## This file is part of the Omics Playground project.
+## Copyright (c) 2018-2022 BigOmics Analytics Sagl. All rights reserved.
+##
+
+#' Activation map plot UI input function
+#'
+#' @description A shiny Module for plotting (UI code).
+#'
+#' @param id
+#' @param label
+#' @param height
+#'
+#' @export
+drugconnectivity_plot_actmap_ui <- function(id,
+                                            label = "",
+                                            height = c(600, 800)) {
+  ns <- shiny::NS(id)
+  info_text <- strwrap("The <strong>Activation Matrix</strong> visualizes the
+                       activation of drug activation enrichment across the
+                       conditions. The size of the circles correspond to their
+                       relative activation, and are colored according to their
+                       upregulation (red) or downregulation (blue) in the
+                       contrast profile.")
+
+  plot_opts <- shiny::tagList(
+    withTooltip(shiny::checkboxInput(ns("dsea_normalize"),
+                                     "normalize activation matrix", FALSE),
+                "Normalize columns of the activation matrix.")
+  )
+  PlotModuleUI(ns("plot"),
+               title = "Activation matrix",
+               label = label,
+               plotlib = "base",
+               info.text = info_text,
+               options = plot_opts,
+               download.fmt = c("png", "pdf", "csv"),
+               width = c("auto", "100%"),
```

Comment:
Please maintain the dimensions of the original plots (that is, if they were adequate)
https://github.com/bigomics/omicsplayground/blob/develop/components/board.drugconnectivity/R/drugconnectivity_server.R#L481

I do not remember if I checked that on your latest PR. Could you please take a quick look?